dnf-behave-tests/dnf/steps/fixtures/httpd.py

```diff
@@ -51,7 +51,14 @@ def do_GET(self):
             self.send_response(self.server._conf['status'])
             self.end_headers()
             return
-        super(LoggingHttpHandler, self).do_GET()
+        if self.server._conf.get("response_url"):
+            if self.server._conf["response_url"] == "/{context.dnf.tempdir}/coprs/testuser/testproject/repo/fedora-31" \
+                                                    "/dnf.repo":
```

Comment:
The path is definitely wrong here. You've just created an arbitrary string which you used as an ID on both sides of the HTTP request, but this is, besiders arbitrary, an unnecessary complication. Use the real path of the request instead. There shouldn't be any scenario-specific constant in this place. Please put the headers which are supposed to be returned in the response into a dict and just retrieve them from the dict based on the real path of the GET request.